docs/modules/develop/pages/guide_migrate_webpacker_module.adoc

```diff
@@ -35,14 +35,79 @@ For images, if they are in `app/packs/images` you could use `image_pack_tag`.
-
-In ActiveAdmin:
-
-- assets are defined in the package.json file, in an entry named `files:`
-- assets are precompiled using Rollup (a lightweigth Js compiler and packer)
-- a generator has been included to copy files to the Rails folder, do a `npm install` and install the npm package. See https://github.com/activeadmin/activeadmin/blob/master/lib/generators/active_admin/webpacker/webpacker_generator.rb[the generator]
-
-It's a good example to follow and get good practices
+As all assets are now compiled using Webpacker without ever loading the Rails or Decidim environment, there are some new conventions how to tell Webpacker about the Decidim module's assets.
+
+To begin with, create a new file named `config/assets.rb` inside your Decidim module.
+
+After this, add the following contents in that file, depending what kind of assets your module provides:
+
+[source,ruby]
+----
+# frozen_string_literal: true
+# This file is located at `config/assets.rb` of your module.
+
+# Define the base path of your module. Please note that `Rails.root` may not be
+# used because we are not inside the Rails environment when this file is loaded.
+base_path = File.expand_path("..", __dir__)
+
+# Register an additional load path for webpack. All the assets within these
+# directories will be available for inclusion within the Decidim assets. For
+# example, if you have `app/packs/src/decidim/foo.js`, you can include that file
+# in your JavaScript entrypoints (or other JavaScript files within Decidim)
+# using `import "src/decidim/foo"` after you have registered the additional path
+# as follows.
+Decidim::Webpacker.register_path("#{base_path}/app/packs")
+
+# Register the entrypoints for your module. These entrypoints can be included
+# within your application using `javascript_pack_tag` and if you include any
+# SCSS files within the entrypoints, they become available for inclusion using
+# `stylesheet_pack_tag`.
+Decidim::Webpacker.register_entrypoints(
+  decidim_foo: "#{base_path}/app/packs/entrypoints/decidim_foo.js",
+  decidim_foo_admin: "#{base_path}/app/packs/entrypoints/decidim_foo_admin.js"
+)
+
+# If you want to import some extra SCSS files in the Decidim main SCSS file
+# without adding any extra stylesheet inclusion tags, you can use the following
+# method to register the stylesheet import for the main application.
+Decidim::Webpacker.register_stylesheet_import("stylesheets/decidim/foo/app")
+
+# If you want to do the same but include the SCSS file for the admin panel's
+# main SCSS file, you can use the following method.
+Decidim::Webpacker.register_stylesheet_import("stylesheets/decidim/foo/admin", group: :admin)
+----
+
+== Component stylesheet migration
+
+In older Decidim versions your components could define their own stylesheet as follows:
+
+[source,ruby]
+----
+Decidim.register_component(:your_component) do |component|
+  component.engine = Decidim::YourComponent::Engine
+  component.stylesheet = "decidim/your_component/your_component"
```

Comment:
I've found that several components are still assigning the `stylesheet` attribute. I understand that this is not needed anymore, nor the `admin_stylesheet` attribute, right? 
I'm thinking of creating a PR removing those attributes and all the assignments in the component manifests.

Reply:
Yes, those attributes no longer have any effect.

If you are planning to remove these attributes from the `Decidim::ComponentManifest` class, I suggest you also create replacement methods for `stylesheet=(_ss)` and `admin_stylesheet=(_ss)` that would throw more explanatory errors for the user guiding them to the migration guide.